### PR TITLE
Feature/ML-211 Implement entered applet password verification

### PR DESCRIPTION
**Pull request by IhorNesterenko99**

### CODE REVIEW

File: src/components/TopBar/Notifications/Notifications.tsx [hunk 1]
```diff
@@ -62,30 +64,34 @@ export const Notifications = ({ alertsQuantity }: NotificationsProps): JSX.Eleme
...
-            {t('alerts')}
-          </StyledLabelLarge>
+          <StyledLabelLarge fontWeight="semiBold">{t('alerts')}</StyledLabelLarge>
         </StyledHeaderLeft>
         <StyledFlexTopCenter>
-          <StyledLabelLarge color={variables.palette.on_surface_variant} fontWeight="semiBold">
-            {`${alertsQuantity} `}
-            {t('unread')}
+          <StyledLabelLarge fontWeight="semiBold" color={variables.palette.semantic.error}>
+            {`${alertsQuantity || ''} `}
```
Comment (kdanilenka32) [inline src/components/TopBar/Notifications/Notifications.tsx:71]: do we need } `} here?

Comment (IhorNesterenko99) [reply src/components/TopBar/Notifications/Notifications.tsx:71]: we need additional space on the end before 'unread'

Comment (kdanilenka32) [reply src/components/TopBar/Notifications/Notifications.tsx]: add before unread  { ' '} and before { alertsQuantity || ''}

**Review by kdanilenka32 — APPROVED** [review]

File: src/components/TopBar/Notifications/Notifications.tsx [hunk 2]
```diff
@@ -62,23 +64,27 @@ export const Notifications = ({ alertsQuantity }: NotificationsProps): JSX.Eleme
...
-            {t('alerts')}
-          </StyledLabelLarge>
+          <StyledLabelLarge fontWeight="semiBold">{t('alerts')}</StyledLabelLarge>
         </StyledHeaderLeft>
         <StyledFlexTopCenter>
-          <StyledLabelLarge color={variables.palette.on_surface_variant} fontWeight="semiBold">
-            {`${alertsQuantity} `}
-            {t('unread')}
-          </StyledLabelLarge>
+          {alertsQuantity > 0 && (
```
[the remark this answers is not in the document]
Comment (IhorNesterenko99) [reply src/components/TopBar/Notifications/Notifications.tsx:70]: we will receive zero (0) it this case

**Pull request merged**